test/javascripts/helpers/widget-test.js

```diff
@@ -1,7 +1,15 @@
 import componentTest from "helpers/component-test";
+const { getProperties } = Ember;
 
-export function moduleForWidget(name) {
-  moduleForComponent(name, `widget:${name}`, { integration: true });
+export function moduleForWidget(name, options = {}) {
+  moduleForComponent(
+    name,
+    `widget:${name}`,
+    Object.assign(
+      { integration: true },
+      getProperties(options, ["beforeEach", "afterEach"])
```

Comment:
`{ beforeEach: options.beforeEach, afterEach: options.afterEach }` is simpler and doesn't require an import.

This isn't a dealbreaker for me but I would prefer it. If you disagree feel free to merge.